### PR TITLE
Build test suite framework for Scala

### PR DESCRIPTION
**Pull request by sfc-gh-mrui**

### CODE REVIEW

**Comment by sfc-gh-mrui** [comment]
https://snowflakecomputing.atlassian.net/browse/SNOW-152805

**Comment by sfc-gh-mrui** [comment]
Below is the README for the test framework. It is included in this PR. List here to let you read it easily. 
https://github.com/snowflakedb/spark-snowflake/blob/dev_mingli2/README-TEST.md

File: .github/docker/check_result.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+export TOTAL_TIMEOUT_IN_SECONDS=3600
+
+echo "GITHUB_RUN_ID=$GITHUB_RUN_ID"
+echo "GITHUB_SHA=$GITHUB_SHA"
+
+echo "Start to downlod dependent libraries"
+curl -s -o scala-library-2.11.12.jar https://repo1.maven.org/maven2/org/scala-lang/scala-library/2.11.12/scala-library-2.11.12.jar
+curl -s -o slf4j-log4j12-1.7.30.jar https://repo1.maven.org/maven2/org/slf4j/slf4j-log4j12/1.7.30/slf4j-log4j12-1.7.30.jar
+curl -s -o slf4j-api-1.7.30.jar https://repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.30/slf4j-api-1.7.30.jar
+curl -s -o snowflake-jdbc-download.jar https://repo1.maven.org/maven2/net/snowflake/snowflake-jdbc/3.12.2/snowflake-jdbc-3.12.2.jar
```
Comment (sfc-gh-bli) [inline .github/docker/check_result.sh:17]: jdbc version can be a variable, and read from sbt file

Comment (sfc-gh-mrui) [reply .github/docker/check_result.sh:17]: > jdbc version can be a variable, and read from sbt file

This JDBC is used to connect to Snowflake to query the result table. It is no necessary to update frequently.

**Comment by sfc-gh-mrui** [comment]
https://github.com/snowflakedb/spark-snowflake/blob/master/README-TEST.md

**Pull request merged**